stdlib/LinearAlgebra/test/uniformscaling.jl

```diff
@@ -460,6 +460,14 @@ end
     @test I(3) == [1 0 0; 0 1 0; 0 0 1]
 end
 
+@testset "dot" begin
+    A = rand(-10:10, 3, 3)
+    λ = rand(-10:10)
```

Comment:
Can you add a check for complex numbers as well?

Reply:
Thanks for catching, definitely got that case wrong